src/Verbs/InitCommand.cs

```diff
@@ -0,0 +1,259 @@
+
+            var node = MakePackageFile(setup);
+
+            Console.WriteLine(node.Element("info").ToString());
+
+            // confirm 
+
+            var confirm = GetUserInput(Resources.Init_Prompt, Defaults.Init_Prompt).ToUpper();
+            if (confirm[0] == 'Y')
+            {
+                // save xml to disk.
+
+                node.Save(packageFile);
+                Console.WriteLine(Resources.Init_Complete);
+                Environment.Exit(0);
+            }
+            else
+            {
+                Environment.Exit(1);
+            }
+        }
+
+        /// <summary>
+        ///  Make a package xml from the options
+        /// </summary>
+        /// <param name="options">options enterd by the user</param>
+        /// <returns>XElement containing the package.xml info</returns>
+        private static XElement MakePackageFile(PackageSetup options)
+        {
+            var node = new XElement("umbPackage");
+
+            var info = new XElement("info");
+
+            var package = new XElement("package");
+            package.Add(new XElement("name", options.Name));
+            package.Add(new XElement("version", options.Version));
+            package.Add(new XElement("iconUrl", ""));
+            package.Add(new XElement("licence", options.Licence,
+                new XAttribute("url", GetLicenceUrl(options.Licence))));
+            
+            package.Add(new XElement("url", options.Url));
+            package.Add(new XElement("requirements",
+                new XAttribute("type", "strict"),
+                new XElement("major", 8),
+                new XElement("major", 0),
+                new XElement("major", 0)));
+            info.Add(package);
+
+            info.Add(new XElement("author",
+                        new XElement("name", options.Author),
+                        new XElement("webiste", options.Website)));
+
+            info.Add(new XElement("readme",
+                new XCData(options.Description)));
+
+            node.Add(info);
+
+            node.Add(new XElement("files"));
+            node.Add(new XElement("Actions"));
+            node.Add(new XElement("control"));
+            node.Add(new XElement("DocumentTypes"));
+            node.Add(new XElement("Templates"));
+            node.Add(new XElement("Stylesheets"));
+            node.Add(new XElement("Macros"));
+            node.Add(new XElement("DictionaryItems"));
+            node.Add(new XElement("Languages"));
+            node.Add(new XElement("DataTypes"));
+
+            return node;
+
+        }
+
+        /// <summary>
+        ///  Workout the URL for the licence based on the string value
+        /// </summary>
+        /// <param name="licenceName">Licence Name (e.g MIT)</param>
+        /// <returns>URL for the licence file</returns>
+        private static string GetLicenceUrl(string licenceName)
+        {
+            // TODO - get licence urls from somewhere?
```

Comment:
We could start simple with Githubs license api? https://developer.github.com/v3/licenses/#list-commonly-used-licenses

With the current Our list there would be a lot that couldn't be found though.

Reply:
yeah  - maybe next release ? - just because it injects a lot of async stuff into a bit that isn't async, and in testing it i just hit the rate limit of unauthorized requests :(

Reply:
I have a branch for it now, but i want to give this a bit more testing https://github.com/KevinJump/Umbraco-Packager-CLI/tree/dev-licenceapi